MonoGame.Framework/Graphics/GraphicsDevice.cs

```diff
@@ -262,6 +278,35 @@ public DepthStencilState DepthStencilState
             }
         }
 
+        internal void ApplyState(bool applyShaders)

```

Comment:
Ok... looks like the reason I did that was to optimize the state changes in that one case of Clear().  Since Clear() is pretty infrequent seems like a pretty tiny win for the extra complication.

We can maybe remove it... not sure yet.